core/src/main/java/io/confluent/kafka/schemaregistry/leaderelector/kafka/ClientConfig.java

```diff
@@ -45,9 +45,8 @@
     CONFIG = new ConfigDef()
         .define(CommonClientConfigs.CLIENT_DNS_LOOKUP_CONFIG,
                 ConfigDef.Type.STRING,
-                ClientDnsLookup.DEFAULT.toString(),
-                in(ClientDnsLookup.DEFAULT.toString(),
-                   ClientDnsLookup.USE_ALL_DNS_IPS.toString(),
+                ClientDnsLookup.USE_ALL_DNS_IPS.toString(),
+                in(ClientDnsLookup.USE_ALL_DNS_IPS.toString(),
```

Comment:
Is this used for anything after the change in KafkaGroupElectorLeader?

Reply:
It looks like it's still used here: https://github.com/confluentinc/schema-registry/blob/master/core/src/main/java/io/confluent/kafka/schemaregistry/leaderelector/kafka/KafkaGroupLeaderElector.java#L123

Reply:
Makes sense. Thanks for checking.